pages/osx/nettop.md

```diff
@@ -0,0 +1,35 @@
+# nettop
+
+> Display dynamic real-time information about the network.
+
+- Start nettop monitoring TCP and UDP sockets from all interfaces:
```

Comment:
```suggestion
- Monitor TCP and UDP sockets from all interfaces:
```
This would make the descriptions a little shorter and less monotonous. That should  be applied everywhere.

Reply:
OK, I will change it